docs/assets/js/search.js

```diff
@@ -23,24 +22,19 @@ function displaySearchResults( results, store ) {
 
       // Show some preview text under each search results item.
       let previewText = '';
-      let itemFieldValue;
-      let field;
+      const searchMatchWordFragment = Object.keys( results[i].matchData.metadata )[0];
+      const searchMatchFields = results[i].matchData.metadata[ searchMatchWordFragment ];
 
-      // Search for the first preview field that is populated.
-      const searchFields = store.fields;
-      for ( const fieldId in searchFields ) {
-        field = searchFields[fieldId];
-        // Exclude certain fields from being considered for the result preview.
-        if ( field === 'title' || field === 'url' ) {
-          continue;
-        }
-
-        itemFieldValue = item[field];
-        if ( typeof itemFieldValue !== 'undefined' && itemFieldValue !== '' ) {
-          previewText = itemFieldValue;
-          break;
-        }
-      }
+      // Remove fields that should never appear as the preview.
+      delete searchMatchFields.id;
+      delete searchMatchFields.title;
+
+      previewText = item[ Object.keys( searchMatchFields )[0] ];
```

Comment:
My linter informs me there shouldn't be a space after this bracket. Now that GitHub Actions is configured for this repo we should add a PR linting job.

Reply:
Fixed! 🦊